### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by ArLEquiN64**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/ArLEquiN64/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
\ No newline at end of file
```
Comment (ArLEquiN64) [inline ReadMe.md]: :sushi: !fix

File: ReadMe.md [hunk 2]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
```
Comment (ArLEquiN64) [inline ReadMe.md]: !fixed

**Comment by ArLEquiN64** [comment]
conversation

**Pull request merged**